ContactDetailsApi.Tests/V2/Gateway/TenureDbGatewayTests.cs

```diff
@@ -56,89 +56,44 @@ private async Task InsertDataIntoDynamoDB(IEnumerable<TenureInformationDb> entit
             }
         }
 
-        private async Task DeleteDataInDynamoDB(IEnumerable<TenureInformationDb> entities)
-        {
-            foreach (var entity in entities)
-            {
-                await _dbFixture.DynamoDbContext.DeleteAsync(entity).ConfigureAwait(false);
-            }
-        }
-
         [Fact]
-        public async Task ScanTenuresReturnsEmptyListWhenNoData()
```

Comment:
I felt this isn't functionality we need to test as we don't need to support the case of the tenure table having no data